### DISCUSSION TITLE
PMS7003 sensor seems to be offline issue

### DISCUSSION THREAD

**Discussion by darecare** [DesconBelgrade/Klimerko#39] [Q&A]
Zdravo,

imam problem kod PMS senzora da ne vraća podatke, dobija se poruka da je offline (prethodno se dobije poruka da je pokušan wake up senzora).
Pokušao sam 2x da odlemim, pa ponovo zalemim žice, promenio sam u drugom pokušaju i GND pin, ali bez uspeha. 
BME senzor mi vraća podatke normalno.

Da li postoji neki način da proverim, da li PMS senzor uopšte radi?
Imate li neki savet, kako da utvrdim gde je problem?

Pozdrav,
Darko

**Comment by epiller** [comment]
Da li se čuje zvuk ventilatora na samom senzoru kada ga spojiš na napajanje (GND i 5V)? Ako ne, vjerovatno je mrtav.
Najbolja varijanta je da probaš sa drugim PMS7003 senzorom ako ga imaš.

**Reply by darecare** [reply]
Hvala na odgovoru. 

U medjuvremenu sam uspeo da povežem PMS senzor da radi, povezao sam na 3.3V pin i GND. U prethodnim tiketima sam pročitao da senzor neće dobro raditi ako je na 3.3, jer ventilator neće raditi. Medjutim u mom slučaju, ventilator radi i pali se prilikom čitanja i onda gasi kako je i definisano u softveru. NodeMCU ploča kao da ne rade pinovi sa desne strane, možda sam ja nešto upropastio prilikom sklapanja. Šaljem sliku kako izgleda.

PMS senzor šalje stalno povećane vrednosti u odnosu na druge lokacije na mapi u Beogradu, vidim da se ventilator i dalje vrti. Da li postoji neki način kako da proverim zašto daje povišene rezultate u odnosu na druge? (okolo su kuće koje lože, možda jedino to može biti uzrok, nisam siguran) 
![1668678543574](https://user-images.githubusercontent.com/25501951/202414452-01882059-6e4f-465e-bc08-c94c4b3f4815.jpg)

**Reply by epiller** [reply]
Jako čudno! Kao što kažeš, ne bi trebalo da radi na 3.3v. Elektronika senzora radi na 3.3v ali zahtijeva 5v zbog ventilatora. Svejedno, ako se okreće na 3.3v, moguće je da je sve okej. Što se tiče očitavanja, pokušaj ga staviti u prostoriju ili kutiju gdje znaš da nema zagadjenja i ostavi ga tu neko vrijeme. Ako nakon par sati i dalje očitavanja budu visoka, moguće da je senzor pokvaren. 
Takodje možeš pokušati da napajaš senzor eksterno sa 5v (ali se pobrini da su GND eksternog napajanja i GND ESP8266 povezani) da vidiš da li pravi ikakvu razliku.
